src/create-envfile.py

```diff
@@ -8,5 +8,18 @@
     if key.startswith("INPUT_ENVKEY_"):
         out_file += key.split("INPUT_ENVKEY_")[1] + "=" + os.environ.get(key) + "\n"
 
-with open("/github/workspace/" + str(os.environ.get("INPUT_FILE_NAME")), "w") as text_file:
-    text_file.write(out_file)
\ No newline at end of file
+# get directory name in which we want to create .env file
+directory = str(os.environ.get("INPUT_DIRECTORY"))
+
+# get file name in which we want to add variables
+# .env is set by default
+file_name = str(os.environ.get("INPUT_FILE_NAME"))
+
+path = "/github/workspace"
+
+if "None" in directory:
+    with open(os.path.join(path, file_name), "w") as text_file:
+        text_file.write(out_file)
+else:
+    with open(os.path.join(path, directory, file_name), "w") as text_file:
+        text_file.write(out_file)
```

Comment:
Instead of having an if/else here, can we just add a default of `''` to the directory? It also appears that you haven't updated the action.yml file, so you'll need to do that to add the new input.

```
  inputs:
    file_name:
      description: 'The filename for the envfile'
      default: '.env'
    directory:
      description: 'The directory to put the envfile in'
      default: ''
```